### ISSUE TITLE
Cant connect with HTTP API

### ISSUE THREAD

**Issue by emersonqueiroz** [rsodre/VLCSyphon#1]
Hi,

I'm trying to control the VLCSyphon player with another sotware, but it seems that the [HTTP API](https://wiki.videolan.org/VLC_HTTP_requests/) is not working with VLCSyphon, i enabled HTTP Requests on VLC, but my requests is not working.

I tried with normal VLC and worked correctly, can you guys please verify?

**Comment by PamelaLiou** [comment]
Yes I would love an answer to this as well, as I stream footage from my raspberry pi cam to  to vlc-player over http

**Comment by rsodre** [comment]
hmm, I have no idea, I just got the source and made Syphon work, did not touch anything else.
Is it used for the remote control also? I remember I tried rc and it did not work either.
